lint/action.yaml

```diff
@@ -5,6 +5,9 @@ inputs:
     required: false
     description: Perform checkout as first step of action
     default: "true"
+  eslint-flags:
```

Comment:
Maybe done later, but we aren't consuming these flags in this commit

Reply:
Will fix

Reply:
Done!